samples/javascript_nodejs/49.qnamaker-all-features/README.md

```diff
@@ -16,7 +16,7 @@ In this sample, we demonstrate
 `- Follow instructions` [here](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/set-up-qnamaker-service-azure) to create a QnA Maker service.
 - Follow instructions [here](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/multiturn-conversation) to create multiturn experience.
 - Follow instructions [here](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/quickstarts/create-publish-knowledge-base) to import and publish your newly created QnA Maker service.
-- Update [appsettings.json](appsettings.json) with your kbid (KnowledgeBase Id), endpointKey and endpointHost. QnA knowledge base setup and application configuration steps can be found [here](https://aka.ms/qna-instructions).
+- Update [.env](.env) with your kbid (KnowledgeBase Id), endpointKey, endpointHost and defaultAnswer. QnA knowledge base setup and application configuration steps can be found [here](https://aka.ms/qna-instructions).
```

Comment:
Same comment here on defaultAnswer - language should probably be clarified to indicate that DefaultAnswer is optional and not mandatory.

Reply:
Added this to readme. "You may also change the default answer by updating `DefaultAnswer` (optional)  field." 
Hope it sounds correct here.